pkg/cmd/server/origin/aggregator.go

```diff
@@ -75,6 +76,7 @@ func (c *MasterConfig) createAggregatorConfig(genericConfig genericapiserver.Con
 		ProxyClientCert:   certBytes,
 		ProxyClientKey:    keyBytes,
 		ServiceResolver:   serviceResolver,
+		ProxyTransport:    utilnet.SetTransportDefaults(&http.Transport{}),
```

Comment:
> So what exactly does this do? I traced ProxyTransport but am having a hard time grokking what is being proxied and by who and why.

It sets our "standard" defaults on a proxy transport that will be used by the aggregator to proxy requests to other API servers after we copy it and mess with dialer options.